client/src/App.js

```diff
@@ -3,14 +3,14 @@ import { Route, Routes } from "react-router-dom";
 import About from "./pages/About";
 import Home from "./pages/Home";
 import Login from "./pages/Login";
-import Registraction from "./pages/Registraction";
+import Registration from "./pages/Registration";
 
 const App = () => (
 	<Routes>
 		<Route path="/" element={<Home />} />
 		<Route path="/about/this/site" element={<About />} />
 		<Route path="/login/this/site" element={<Login />} />
-		<Route path="/registraction/this/site" element={<Registraction/>} />
+		<Route path="/registraction/this/site" element={<Registration/>} />
```

Comment:
maybe add the path as /student-registration

Reply:
otherwise looks good

Reply:
Hi @mfukechristian , not sure if you are aware, but we have approved your changes, but you have to resolve the conversations, fix the es lint as well as the branch conflicts and then push again as we need to review again

Reply:
yea I saw it I have been struggling to  merge  can we do it tomorrow please